plugin/healthcheck.go

```diff
@@ -30,20 +32,22 @@ func (*HealthcheckPlugin) Version() string {
 
 // Init initializes plugin, satisfying the plugin.Plugin interface.
 func (*HealthcheckPlugin) Init(env *plugin.Environment) error {
-	_port := os.Getenv(portEnvVar)
-	if _port != "" {
-		port = _port
+	envPort := os.Getenv(portEnvVar)
+	if envPort != "" {
+		port = envPort
 		return nil
 	}
 
 	cfg, ok := env.Config.(map[string]interface{})
-	if ok {
-		_port, ok := cfg["port"].(string)
-		if ok {
-			port = _port
-			return nil
-		}
+	if !ok {
+		return errors.New("env config not in known format")
```

Comment:
I don't think we want to error here. We can just use default port and log warning?

I'm guessing error here would stop the process?